### PR TITLE
Downgrade rake to 13.0.1

### PR DESCRIPTION
**Pull request by hennevogel**
Also bring back the test for the rake version and make sure the correct alternative is set on the appliance

### CODE REVIEW

File: dist/obs-bundled-gems.spec [hunk 1]
```diff
@@ -105,7 +105,12 @@ bundle config build.nokogiri --use-system-libraries
 
 bundle --local --path %{buildroot}%_libdir/obs-api/
 
+# Make sure rake and rack in Gemfile.lock match the versions from the
+# rubygem-rack and ruby2.7 packages
+# otherwise Passenger wont start the app
+
 test -f %{buildroot}%_libdir/obs-api/ruby/2.7.0/gems/rack-%{rack_version}/rack.gemspec
+test -f %{buildroot}%_libdir/obs-api/ruby/2.7.0/gems/rack-%{rake_version}/rack.gemspec
```
Comment (M0ses) [inline dist/obs-bundled-gems.spec:113]: Is this line correct? rack != rake. it looks like you copied the line and changed only rack_version -> rake_version but not changed the gem name.

Comment (hennevogel) [reply dist/obs-bundled-gems.spec]: For sure not! Thanks

**Comment by hennevogel** [comment]
Test building in https://build.opensuse.org/package/show/home:hennevogel:branches:OBS:Server:2.10:Staging/obs-server

**Comment by hennevogel** [comment]
@M0ses can you have a look again?

**Review by rubhanazeem — APPROVED** [review]

**Pull request merged**